central/complianceoperator/v2/report/manager/complianceReportgenerator/consts.go

```diff
@@ -0,0 +1,13 @@
+package complianceReportgenerator
+
+const (
+	defaultEmailBodyTemplate = "{{.BrandedPrefix}} has identified {{.ComplianceStatus}} profile checks for clusters scanned by your \n" +
+		"schedule configuration parameters. The attached report lists those checks and associated details to aid with remediation. \n" +
```

Comment:
{{.BrandedPrefix}} has scanned your clusters for compliance with profiles in your scan configuration. The attached report lists the checks  performed and their corresponding details to aid with remediation.

Here's an idea for you: .ComplianceStatus will be non compliant over 90% of the time hence it might be good to avoid that in the wording all together. The email body should be informational saying: here's the report for x profiles accross y clusters. go look :-)

Suggestion: We could enlist the doc team's assistance with the wording.